bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java

```diff
@@ -2194,15 +2198,18 @@ Monitor getMonitor (long hmonitor) {
 	int autoscaleZoom;
 	if (DPIUtil.isAutoScaleOnRuntimeActive()) {
 		autoscaleZoom = DPIUtil.getZoomForAutoscaleProperty(result);
+		monitor.setBounds(getMonitorBoundsInPointsInDisplayCoordinateSystem(boundsInPixels, autoscaleZoom));
+		monitor.setClientArea(getMonitorBoundsInPointsInDisplayCoordinateSystem(clientAreaInPixels, autoscaleZoom));
 	} else {
 		autoscaleZoom = DPIUtil.getDeviceZoom();
+		monitor.setBounds(DPIUtil.scaleDown (boundsInPixels, autoscaleZoom));
+		monitor.setClientArea(DPIUtil.scaleDown (clientAreaInPixels, autoscaleZoom));
```

Comment:
The original code was using `autoScaleDown`. I thought this PR was supposed to leave the original code untouched and only change the case where `DPIUtil.isAutoScaleOnRuntimeActive() == true` ?

Reply:
Changed